dag/chartbook.yml

```diff
@@ -44,6 +44,33 @@ steps:
   data://garden/chartbook/2024-08-05/katic_leigh_2015_australia:
     - data://meadow/chartbook/2024-08-05/katic_leigh_2015_australia
 
+  # Household income and wealth, Australia, Australian Bureau of Statistics (live version)
+  data://meadow/abs/2024-08-06/household_income_and_wealth_australia:
+    - snapshot://abs/2024-08-06/household_income_and_wealth_australia.zip
+  data://garden/abs/2024-08-06/household_income_and_wealth_australia:
+    - data://meadow/abs/2024-08-06/household_income_and_wealth_australia
```

Comment:
I didn't think it was the right option in this case, because the only reason I have both (2017-18 and this one) is because the former has additional data and I want to keep this one as the live version (the one to be updated)